git/02.development-workflow.md

```diff
@@ -18,6 +18,10 @@ supercede general best practices.
 Start by creating a branch from the [release branch](07.release-branching.md)
 and switching to it.
 
+```
+$ git checkout -b feature_branch_name
+```
```

Comment:
Your idea on the merge strategy PR of having examples at the end would work well here, too, I think. I'll add a commit to accomplish this in a bit…